### DISCUSSION TITLE
「再睡5分鐘」對身體有害！美研究揭手機鬧鐘「賴床平均時間」

### DISCUSSION THREAD

**Discussion by HuangHsinTzu** [1101-GS4719-C/Python-Community-GS4719-C#1036] [General]
大家平常都有設貪睡鬧鐘的習慣嗎
本人就是那種要賴床才會爽的人 嘻嘻
但其實老實說賴床真的是一件蠻沒意義的事
因為你並不會因為貪睡那幾分鐘變得更有精神
而且反而會更想睡 更傷腦 
文章裡提到甚至會為心血管帶來壓力
如果遇到重要日子無法賴床的時 鬧鐘一響我就會在心裡默數2秒然後馬上彈起來
避免自己再睡著
這招真的很有用!
推薦給大家
2023新年新希望: 改掉賴床的習慣!
文章連結👇
https://newtalk.tw/news/view/2022-12-20/849356

**Comment by bonny123865** [comment]
喔買尬 我都不知道原來突然被鬧鐘嚇醒會對身體有刺激
我都是三到四分鐘設定一個鬧鐘，但是至少不太會賴床?!
我的習慣是默數30秒，這個30秒不一定要按照時鐘的那種
雖然可以拉長時間，但是至少自己是清醒的就足夠了!
我都是靠吃東西來醒腦的哈哈
每年新希望：準時起床

**Comment by moonnight3503** [comment]
哈哈，我是那個超過賴床平均數的其中一位
我依賴床大概就是半小時起跳吧?
但我中間都不會反覆起床，我會直接忽略那個鬧鐘哈哈哈
但如果是重要的日子確實是不會賴床呢
心裡會有一個警告說:「今天絕對不能賴床！」
哈哈哈

**Comment by Valentinaa0214** [comment]
這根本是我🥲
我鬧鐘都是從七點半響到九點，每五分鐘一個，
最後十點才起床，早九的課十一點才到教室，
每天上床前都信誓旦旦的跟室友說明天絕對會早起吃早餐，
整學期一次也沒做到....
但大學生真的永遠都睡不飽阿！！！

**Comment by Yunhans** [comment]
我之前都會按好幾次再睡五分鐘，但被室友抗議之後就開始不按了。

**Comment by BlahFish1006** [comment]
我都會每隔5到10分鐘再設一個鬧鐘，有時候不是不想起，是會累到聽不到QQ
或是身體反射動作直接在無意識中就把鬧鐘關掉了，真的超級危險阿；)
但確實每次賴床後反而精神都不會比較好，原來還會對身體有害！

還是新年新希望：每天睡飽飽 精神好 自然清醒最舒服

**Comment by ghost** [comment]
我覺得不好歸不好，但沒辦法，有時就是會忙到很晚，但明早的某時段必須要起床，為了以防鬧鐘在該時段叫了卻沒醒，只好在該時段以前多設幾個鬧鐘，來確保能確實醒來，總之只能說是身不由己，不得以而為之。總結，誰不想每天都能睡到自然醒，但沒辦法啊

**Comment by manipulateed** [comment]
原來賴床也會成為影響身體健康的兇手之ㄧ阿，但究竟要怎麼做才不會賴床呢，早睡的人就不會賴床嗎，既然賴床會有害，那是不是就能朝這個 方向去發明一定能叫醒自己的工具。

**Comment by Floriaand** [comment]
可是我覺得賴床的那幾分鐘是我一天中最快樂的，對我來說是會比較有精神耶，而且我真的超容易有生理時鐘，只要個三四天就會習慣多睡的那幾分鐘然後會自己忽然醒來，很準時。我身邊的朋友都覺得我很厲害可以很早起床，其實都是生理時鐘。

**Comment by dabbie109303501** [comment]
雖然賴床會對身體帶來危害
但...在賴床的5~10分鐘內
我體會到了人生的一大樂趣...🥺

而且如果不多設幾個鬧鐘 萬一我迷迷糊糊按掉了一堆最後遲到怎麼辦QAQ

**Comment by s96015welson** [comment]
我自己也是常常會賴床，雖然不至於到會上課遲到，但晚睡對生活品質影響確實很大，賴床那幾分鐘根本補不回來。健康的生活作息才是改善賴床的最佳方法。

**Comment by Kotori0501** [comment]
我平常是沒用賴床的習慣，只要鬧鐘一響，我會先躺在床上睜眼1分鐘再起床，因直接起床其實也會對心血管疾病造成一定的負擔，所以一定要比先讓自己比較清醒再爬起來會比較好。

**Comment by wenchi9023** [comment]
幸好我不是會賴床的人，每次鬧鐘一響就會準時起床，看完這篇文章後發現原來自己準時的習慣都有默默地回饋在身體上呀

**Comment by ru0372572** [comment]
其實會有意識地知道第一個鬧鐘關掉之後還會有第二個，所以就不會起來哈哈
有時候還會關掉最後一個想說等一下就會起來，然後再過了一個小時:)
話說看標題第一眼還以為是飲料店(?

**Comment by WEI-CHEN-CHUN** [comment]
早八的課要設7.三個鬧鐘，然後還是沒起來，關掉就繼續睡，有時候真的是太晚睡了，
作息正常固定真的很重要

**Comment by AnAnistired** [comment]
再睡五分鐘對身體有害，那再睡十分鐘就可以啦！好啦其實我也是這樣常常賴床，但我覺得這問題的根本原因是太晚睡，因為如果我今天早點睡覺，隔天就會很早自然醒，根本就不會有賴床的問題😆

**Comment by Guagua1234** [comment]
每次賴床完之後都會覺得特別疲憊，原來是這個原因。
也常常被鬧鐘嚇醒原來這樣會對身體造成負擔，
看來要改進的地方還很多呢
我發現最好的鬧鐘還是人體鬧鐘。

**Comment by delia07152** [comment]
如果設一個以上的鬧鐘，然後間隔五分鐘的話，那我可能第一次醒來就會開始在心中倒數五分鐘，然後越靠近五分鐘，就會越來越緊張，因為很怕會被鬧鐘嚇到，所以之後我就索性就設一個，不過很偶爾會無意識的關掉鬧鐘，希望之後不會在段考前發生這種事！

**Comment by Pietra1226** [comment]
一開始看到標題還以為是飲料店...。

**Comment by Sylvie0601** [comment]
居然如此!
但我還是超愛賴床，應該是說我根本起不來哈哈，我鬧鐘都10分鐘定一個，看完這篇文章後希望2023的自己不要再賴床了哈哈，本來身體就因為熬夜更不好了，結果賴床也會讓身體變差(哭

**Comment by Twenty0522** [comment]
我通常只會設定一個鬧鐘，依照睡眠週期睡覺，比較容易從睡夢中醒來，也比較不容易睡回去，所以不會每五分鐘就設定一個鬧鐘，但睡醒後會在床上躺幾分鐘緩衝再起床，如果是要提醒自己真的該下床了我會想多設定一個鬧鐘。

**Comment by yuxinlin93** [comment]
我個人在平常上課日都會小賴床一下，就是稍微清醒一下把鬧鐘關掉，然後慢慢起床，比較不會有那種馬上從深層睡眠被挖起來的可怕感覺，但我比較不會設太多鬧鐘，一位這樣我就會覺得"沒關係 我可以在睡一下 因為等一下還會有另一個鬧鐘把我叫起來" 通常這樣想就會越睡越沉，下一次醒來可能就是中午的事了

**Comment by harrylee1971** [comment]
今日系上教授說一次睡眠的循環為90分鐘，如文中提到，這樣醒了又睡，睡了又起來重睡的行為真的會造成睡眠品質不佳
要就起床不然就繼續睡到飽!!!

**Comment by a45320614** [comment]
我個人是一個相當嚴重的賴床患者，早9的課可以賴到11點，我對鬧鐘的免疫力可以說是相當的高。為了解決賴床問題，我有特別下載一款必須要算出數學題目才可以關掉鬧鐘的App，而且關掉後隔5分鐘它還會檢查你是不是真的起床了，如果沒有及時按下螢幕上的按鈕，它就會無限輪迴一直響，但常常還是我室友全醒了然後我還沒醒XD

**Comment by HsuChen88** [comment]
默數10秒我試過，但我好像是時空旅人，明明是8點的鬧鐘，數10秒後起床，手機顯示12點。
想學時空旅行的人以下開放報名。

**Comment by Jzd314** [comment]
通常我會設3個鬧鐘，第一個鬧鐘讓我張開眼睛，第二個讓我開始清醒，第三個讓我離開床上

**Comment by 110302550** [comment]
我通常會早設個一個小時，然後每5分鐘設一個鬧鐘，原來那麼慘喔，要改了

**Comment by woohsin** [comment]
前幾天根本不是再睡五分鐘的問題，鬧鐘被我調成靜音，基本上等於沒功能，所以直接睡到自然醒，真心崩潰...

**Comment by YANRONG0123** [comment]
我一直都很想要鬧鐘一響就馬上起床，但這取決於前一天我的作息，也許太累，但睡眠時數短缺，會導致我直接賴床，而且我鬧鐘是沒有設定貪睡功能，所以賴下去，幾乎就是直接翹課了，有一次還差點趕不上公車回家。

**Comment by YifenLin** [comment]
完蛋，這完全我，每天晚上睡前都想說要改掉這個壞習慣，但到了早上還是忍不住賴床繼續睡，畢竟要起床真的很困難，尤其是冬天賴床的狀況更嚴重。另外，因為宿舍是睡上舖的關係，要爬下樓梯又增加了我與下床的距離Wwwww

**Comment by hbh10601** [comment]
這我還真的從來不知道，雖然我都很晚睡，不過我還是不會賴床，甚至還是叫室友起床的那個人，但這讓我開始想要慢慢改變睡眠習慣了。

**Comment by tzurou** [comment]
我自己是習慣只設一個鬧鐘，因為我覺得多設幾個鬧鐘會讓自己更難爬起來，如果鬧鐘響了還是很想睡的話就會當下設一個5分鐘或10分鐘的鬧鐘，通常第二個鬧鐘響後我一定會起床，但就是偶爾會忘記再設一個就睡過頭了QQ

**Comment by 110101501** [comment]
以前會每五分鐘設一個鬧鐘，但後來發現成效很差，也蠻影響自己的睡眠品質，就不這樣執行了。
自己現在習慣設定兩個鬧鐘，一個先喚醒自己讓自己知道已經是早上了，順便賴個床。二十分鐘後會再有一個鬧鐘告訴自己真的要起床了。

**Comment by willychiuchiu** [comment]
我覺得賴床是一種習慣，以前在家真的很會賴，想說反正有爸媽會把我挖起來所以就很放心的賴床，但自從自己出來生活後，發現就算賴到考試結束了都沒有人會叫你，睡過幾次考試就知道事情的嚴重性了，於是開始想辦法改變，強制自己每天要在兩個鬧鐘內起床，現在幾乎都會起床了（除了有時候真的很累）

**Comment by yxin214** [comment]
以前都不知道賴床會影響健康！我是一個超會賴床的人，多睡那一、兩分鐘也覺得滿足。以前嘗試過版主提供的方法，就是聽到鬧鐘後默數個幾秒一定要從床上爬起來，覺得滿有效的，但就是缺少了那份賴床的滿足感哈哈哈。知道賴床會影響健康後可能還是得乖乖改掉這習慣了。

**Comment by happy246810** [comment]
我本身會設定兩個鬧鐘，一個是我要起床的時間，一個5分鐘後的鬧鐘。因為鬧鐘響的時候，我按掉之後，我會繼續躺著回神一下。而設定一個5分鐘後的鬧鐘是為了以防萬一我睡過頭，看完報導後，我以後聽到第一個鬧鐘就會醒了。

**Comment by yyujimmy** [comment]
貪睡確實是許多人都會遇到的問題，但你提到的點子確實很有用！為了避免貪睡對身體造成的壓力，改掉貪睡的習慣確實是一個很好的新年新希望。我們的身體需要充足的睡眠以保持健康和活力，而過度的貪睡反而可能會影響我們的生活品質。

我們可以試著設定一個合適的起床時間，並盡量遵守它，即使在週末或假期也是如此。此外，建立一個健康的睡眠習慣，如避免在睡前使用電子設備、保持良好的睡眠環境等，也有助於改善睡眠質量，減少貪睡的需求。

感謝分享這篇文章，讓我們一起努力改掉貪睡的習慣，迎接更健康、更有活力的新的一年！

**Comment by KH1126** [comment]
我自己習慣設五六個鬧鐘，起床時間前面一小時和半小時的鬧鐘提醒我時間快到了，但我還可以賴個床。
後面大概每五分鐘會有一次鬧鐘，確保我會起床。
感覺我大學舍友容忍我有時候太賴沒把鬧鐘關掉。

**Comment by Mitochondrionjessica** [comment]
我自己是會設定兩個鬧鐘，間隔三分鐘。對我而言第一個鬧鐘響後我會醒來，但會有起床氣所以需要三分鐘的緩衝時間，然後躺在床上想今天的規劃跟行程，有點像一天正式開始前跟自己對話的小儀式，調整好自己的狀態在起床。

**Comment by jim1743** [comment]
平時我是不會賴床的人，鬧鐘一響，我都會立馬起來，深怕今天早上的課會遲到，因此我只會設一個鬧鐘，但若是假日，即使鬧鐘響了，我還是會繼續睡，因為早上沒有事情需要這麼早起，因此假日大多會睡到自然醒，並且也不設鬧鐘，讓自己的睡眠品質好一些。

**Comment by cassy0201** [comment]
我也是會設兩三個鬧鐘然後拖到最後一個才起床的人，中間那10分鐘說實話也睡不安穩，而且會越睡越累，最近在試著改掉這個換習慣了

**Comment by yubo0oo** [comment]
這是我的鬧鐘
天天被驚醒
幾乎沒有賴床
推薦給各位
https://www.youtube.com/watch?v=ZGPDfFLEGbU&pp=ygUK5LiB54m5IEZCSQ%3D%3D
